### PR TITLE
Starting the Lookout Style Guides

### PR DESCRIPTION
**Pull request by polynomial**
Removing placeholders and slightly modifying the README to be more
org specific.

### CODE REVIEW

File: perl/README.md [hunk 1]
```diff
@@ -1 +0,0 @@
-Don't use perl.
```
Comment (mbbx6spp) [inline perl/README.md:1]: I think this has to stay in tact. :)

**Comment by mbbx6spp** [comment]
I agree with all other changes however.

File: perl/README.md (hunk 1 above)
Comment (mbbx6spp) [inline perl/README.md:1]: I will make a slight revision:

`Don't use Perl for new development. Only maintain existing internal or third-party Perl. Use the following for styleguide of Perl code: http://perldoc.perl.org/perlstyle.html`

**Comment by polynomial** [comment]
made all changes, merging.

**Pull request merged**